myapp/spec/factories/tasks.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :task do
+    title { "ThisIsTitle1" }
+    description { "ThisIsDescription1" }
+  end
+end
```

Comment:
```suggestion
# frozen_string_literal: true

FactoryBot.define do
  factory :task do
    title { 'ThisIsTitle1' }
    description { 'ThisIsDescription1' }
  end
end
```

I recommend checking with rubocop to follow the coding guidelines for Rakuma.

```
❯ bundle exec rubocop spec/factories/tasks.rb  
Inspecting 1 file
C

Offenses:

spec/factories/tasks.rb:1:1: C: [Correctable] Style/FrozenStringLiteralComment: Missing frozen string literal comment.
FactoryBot.define do
^
spec/factories/tasks.rb:3:13: C: [Correctable] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
    title { "ThisIsTitle1" }
            ^^^^^^^^^^^^^^
spec/factories/tasks.rb:4:19: C: [Correctable] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
    description { "ThisIsDescription1" }
```

Reply:
Ok, I just fixed it using single quote.
Yes, actually I've seen a lot about this single/double quote warning by rubocop. Is there any place I can check Rakuma coding guide line or common rubocop config(rubocop.yaml)?

Reply:
Thanks for sharing the config. I will set it on my branch.